### PR TITLE
Update sample data fixture file

### PR DESCRIPTION
**Pull request by Maffooch**

### CODE REVIEW

**Comment by madchap** [comment]
Re-ordering? What's the PR about?

**Comment by Maffooch** [comment]
The sample data file would not get loaded with the way it was ordered before. The tags were being referenced before definition. This resolves that issue so that the file is actually worth something again.

**Comment by valentijnscholten** [comment]
should we add a quick unit test that uses this fixture just to make it loads well?

**Comment by valentijnscholten** [comment]
nice, now I will know if I break it again :-)

**Review by valentijnscholten — APPROVED** [review]

File: dojo/fixtures/defect_dojo_sample_data.json [hunk 1]
```diff
@@ -40633,26 +40653,6 @@
         "meta_encoded": "{}"
     }
 },
-{
```
Comment (damiencarol) [inline dojo/fixtures/defect_dojo_sample_data.json:40636]: Why reordering that @madchap ?

**Comment by valentijnscholten** [comment]
I think you can just do 'fixture = blabla.json' and django will load it for you. But explicit load also works

**Review by madchap — APPROVED** [review]

**Pull request merged**